src/main/java/teammates/common/util/Const.java

```diff
@@ -972,6 +976,10 @@ private Const() {
                 "/page/instructorFeedbackRemindParticularStudents";
         public static final String INSTRUCTOR_FEEDBACK_PUBLISH = "/page/instructorFeedbackPublish";
         public static final String INSTRUCTOR_FEEDBACK_UNPUBLISH = "/page/instructorFeedbackUnpublish";
+        public static final String INSTRUCTOR_FEEDBACK_PUBLISH_EMAIL_PARTICULAR_STUDENTS_PAGE =
+                "/page/instructorFeedbackPublishEmailParticularStudentsPage";
+        public static final String INSTRUCTOR_FEEDBACK_PUBLISH_EMAIL_PARTICULAR_STUDENTS =
+                "/page/instructorFeedbackPublishEmailParticularStudents";
```

Comment:
`INSTRUCTOR_FEEDBACK_PUBLISH_EMAIL_PARTICULAR_STUDENTS_PAGE ` needs to validate origin as well? Because I see that `INSTRUCTOR_FEEDBACK_REMIND_PARTICULAR_STUDENTS_PAGE` is not validated.

Reply:
Oh yea no need to protect the `_PAGE` as it's non-mutating. But do protect the other one as it sends emails out.